packages/super-input/src/SuperInput/extensions/autocomplete/index.ts

```diff
@@ -79,7 +79,7 @@ const globsCompletions = (globs: any) => {
         .replace(']', '')
         .split(/\.|\?\./)
 
-      const from = /[\.\[]/.test(nodeBefore.name) ? nodeBefore.to : nodeBefore.from
+      const from = /[.[]/.test(nodeBefore.name) ? nodeBefore.to : nodeBefore.from
```

Comment:
does this evaluate the dot literally since it's inside `[]`?

Reply:
can we remove the `\` on the `[` also?

Reply:
Yeah, exactly. I didn't catch this with my own eyes or anything, webpack was throwing a warning about it being unnecessary so I just cleaned it up.